lib/tasks/skin_tasks.rake

```diff
@@ -148,6 +148,8 @@ namespace :skins do
 
   desc "Load site skins"
   task(:load_site_skins => :environment) do
+    settings = AdminSetting.first
+    settings.update(default_skin_id: Skin.default.id, last_updated_by: Admin.first.id) if settings.default_skin_id.nil?
```

Comment:
To get around the validation that requires assigning every update to an admin, you can do this:

https://github.com/otwcode/otwarchive/blob/3785907b55d3d844e4d73deb54f0a8f8c3d9a44b/app/models/admin_setting.rb#L44-L45

instead of picking a random admin.